examples/list-boards/index.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-env node */
+const Libhoney = require("libhoney");
+const process = require("process");
+
+const honey = new Libhoney({
+  apiHost: process.env.HONEYCOMB_API_HOST,
+  apiKey: process.env.HONEYCOMB_API_KEY
+});
+
+honey.boards
+  .list()
```

Comment:
How do you imagine this evolving to support filtering the list of boards? Would it be `honey.boards.list({filter: 'mine'})` or similar?

In a Ruby API I could imagine it looking something like

```ruby
boards = honeycomb.boards.mine
# as a shortcut for
#      boards = honeycomb.boards.all(filter: 'mine')
boards.each {|board| puts board.name }
```